draft-ietf-httpbis-http2bis.xml

```diff
@@ -1085,6 +1085,19 @@
             critical frames, such as <xref target="WINDOW_UPDATE" format="none">WINDOW_UPDATE</xref>, are not read and acted upon.
           </t>
         </section>
+        <section anchor="FlowControlPerformance">
+          <name>Flow Control Performance</name>
+          <t>
+            If an endpoint cannot ensure that its peer always has available flow control window space that is greater than the peer's
+            bandwidth-delay product on this connection, its receive throughput will be limited by HTTP/2 flow control. This will result
+            in degraded performance.
+          </t>
+          <t>
+            Sending timely <xref target="WINDOW_UPDATE" format="none">WINDOW_UPDATE</xref> frames can mitigate this bottleneck. Endpoints will
```

Comment:
QUIC says "improve performance", which I think is better than "mitigate this bottleneck".  bottleneck is a colloquialism that might be hard to understand, and mitigation isn't what you want (you want avoidance in that case).